vtex/toolbelt
# Downgrade to use opn

#### What is the purpose of this pull request?
Downgrade to use `opn` due to issues on `vtex login`.

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/deprecation-check && \
yarn && yarn watch
```
Check `vtex login gc-ene4380`
Check `vtex login storecomponents`

This need to be checked on wsl, windows, linux and MacOS. I will check on WSL, WIndows and Linux.


#### Types of changes

#### Chores checklist
